react/src/components/QrReaderOverlay/QrReaderOverlay.tsx

```diff
@@ -164,6 +167,24 @@ function QrReaderOverlay({
   );
 
   const [boxLabelInputValue, setBoxLabelInputValue] = useState("");
+  const [boxLabelInputError, setBoxLabelInputError] = useState("");
+
+  const onBoxLabelInputChange = useCallback(
+    (value: string) => {
+      if (!value) {
+        // remove error for empty form field
+        setBoxLabelInputError("");
+      } else if (value.length < 6) {
+        setBoxLabelInputError("Please enter more than 6 digits.");
```

Comment:
Do we accept 6-digit numbers or not? The condition says yes but the message says no.

Reply:
@HaGuesto okay I understood how it works: it's a live validation of the typed input. It should be of 6 digits or more. If it's less, an error message is shown. But it should say "Please enter at least 6 digits."

Reply:
@pylipp I have also changed the validation message based on your suggestion :)